app/renderer/actions/Session.js

```diff
@@ -310,8 +310,9 @@ export function newSession (caps, attachSessId = null) {
           });
           return;
         }
-        desiredCapabilities.testdroid_source = 'appiumdesktop';
-        desiredCapabilities.testdroid_apiKey = accessKey;
+        desiredCapabilities['bitbar:options'] = {};
```

Comment:
this could be simplified to
```js
desiredCapabilities['bitbar:options'] = {
  source: 'appiumdesktop',
  apiKey: accessKey,
}
```

Reply:
looking at most other object inits in this file, I assumed this is preffered